cypress/support/step-definitions/common-steps.js

```diff
@@ -22,6 +22,10 @@ import {
 import { dialogTitle } from "../../locators/dialog";
 import DEBUG_FLAG from "..";
 import { pagerSummary } from "../../locators/pager";
+import {
```

Comment:
Why were Button-specific steps added to this common file? Were they required for other components that had Button embedded?

Reply:
They are testing for the enable and disable to different types of button, I don't think it's needed for other components.

Reply:
I think those steps are also used in `splitButton` or `multiActionButton`